gtsfm/scene_optimizer.py

```diff
@@ -381,11 +471,15 @@ def create_computation_graph(
                 camera_intrinsics_graph[i1],
                 camera_intrinsics_graph[i2],
                 use_intrinsics_in_verification,
+                gt_relative_pose,
             )
             i2Ri1_graph_dict[(i1, i2)] = i2Ri1
             i2Ui1_graph_dict[(i1, i2)] = i2Ui1
             v_corr_idxs_graph_dict[(i1, i2)] = v_corr_idxs
 
+            auxiliary_graph_list.append(rot_error)
+            auxiliary_graph_list.append(unit_tran_error)
+
```

Comment:
If the API change I suggested is made, we can access these and append them to the graph only if a condition is met (I think thats if gt_relative_pose is not None).